lang/strings_english.txt

```diff
@@ -1592,6 +1592,7 @@ $s_month_december = 'December';
 
 # Timeline Feature Strings
 $s_timeline_issue_created = '<span class="username">%1$s</span> created issue <span class="issue_id">%2$s</span>';
+$s_timeline_issue_file_added = '<span class="username">%1$s</span> added attachment(s) to issue <span class="issue_id">%2$s</span>';
```

Comment:
If we go with entry per file, then use `attachment`.

Reply:
Fixed